js-exercises/sum-even-args/sumEvenArgs.js

```diff
@@ -0,0 +1,16 @@
+const sumEvenArgs = (...args) => {
+  
+  if(typeof args[0] !== 'number')
+  	return 0;
+
+  let sumOfEvenArgs=0;
+
+  for(let i=0; i<=args.length;i++){
```

Comment:
As of Tuesday's session we should use map/filter as guided by instructor for readable and predictable code.